src/CTA.Rules.Config/Utils.cs

```diff
@@ -267,5 +301,50 @@ public static void DownloadFilesToFolder(string s3Bucket, string targetFolder, L
                 }
             });
         }
+
+        public static IEnumerable<string> GetProjectPathsFromSolutionFile(string solutionPath)
+        {
+
+            IEnumerable<string> projectPaths = null;
+            try
+            {
+                SolutionFile solution = SolutionFile.Parse(solutionPath);
+                projectPaths = solution.ProjectsInOrder.Where(p => AcceptedProjectTypes.Contains(p.ProjectType)).Select(p => p.AbsolutePath);
+            }
+            catch (Exception ex)
+            {
+                LogHelper.LogError(ex, $"Error while parsing the solution file {solutionPath}");
+            }
+
+            return projectPaths;
+
+        }
+
+        public static HashSet<SolutionProjectType> AcceptedProjectTypes = new HashSet<SolutionProjectType>()
+        {
+            SolutionProjectType.KnownToBeMSBuildFormat,
+            SolutionProjectType.WebDeploymentProject,
+            SolutionProjectType.WebProject
+        };
```

Comment:
Similar changes were made to codelyzer: 
https://github.com/aws/codelyzer/pull/127/files
https://github.com/aws/codelyzer/pull/139/files

Can we reuse that code?